src/cfclient_config.erl

```diff
@@ -96,15 +94,11 @@ normalize(Config0) ->
   Config1 = maps:from_list(Config0),
   Config2 = maps:merge(defaults(), Config1),
   Config = normalize_config(Config2),
-  case maps:get(name, Config) of
```

Comment:
This ensures that the "default" instances have the `default` prefix for ets tables whereas before it didn't have the prefix.